test/upgrade/prober/configuration.go

```diff
@@ -51,11 +51,12 @@ var eventTypes = []string{"step", "finished"}
 // Config represents a configuration for prober.
 type Config struct {
 	Wathola
-	Namespace     string
-	Interval      time.Duration
-	FinishedSleep time.Duration
-	Serving       ServingConfig
-	FailOnErrors  bool
+	Namespace       string
+	Interval        time.Duration
+	FinishedSleep   time.Duration
+	Serving         ServingConfig
+	FailOnErrors    bool
+	IgnoreDuplicate bool
```

Comment:
Maybe lets name that `OnDuplicate DuplicateAction` with `DuplicateAction=warn` by default.

```golang
type DuplicateAction string
const (
  Silence DuplicateAction = "silence"
  Warn DuplicateAction = "warn"
  Error DuplicateAction = "error"
)

```

Reply:
Changed as suggested.